Content.Server/Connection/Whitelist/WhitelistCondition.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+using Robust.Shared.Network;
+
+namespace Content.Server.Connection.Whitelist;
+
+[ImplicitDataDefinitionForInheritors]
+[MeansImplicitUse]
+public abstract partial class WhitelistCondition
```

Comment:
I need a doc comment explaining what these conditions are and how they are used (reference to prototype).